File: Java/069. Eq2Plus/Eq2Plus.java
import java.util.Scanner;
public class Eq2Plus
{
    public static void main(String[] args)
    {
        Scanner scan = new Scanner(System.in);
        double A = scan.nextDouble();
        double B = scan.nextDouble();
        double C = scan.nextDouble();
        double delta = B * B - 4 * A * C;
        if (A == 0 || delta < 0)
        {
            System.out.println("nao ha raiz real");
        } 
        else if (delta == 0)
        {
            double r = -B / (2 * A);
            System.out.printf("%.4f\n", r);
        }
        else
        {
            double r1 = (-B + Math.sqrt(delta)) / (2 * A);
            double r2 = (-B - Math.sqrt(delta)) / (2 * A);
            System.out.printf("%.4f", r1);
            System.out.print(" ");
            System.out.printf("%.4f\n", r2);
        }
        scan.close();
    }
}

File: C++/069. Eq2Plus/Eq2Plus.cpp
#include <iostream>
#include <cmath>
#include <iomanip>
int main ()
{
    double A, B, C;
    std::cin >> A >> B >> C;
    double delta = B * B - 4 * A * C;
    std::cout << std::setprecision(4) << std::fixed;
    if (A == 0 || delta < 0)
    {
        std::cout << "nao ha raiz real" << std::endl;
    }
    else if (delta == 0)
    {
        std::cout << -B / (2 * A) << std::endl;
    }
    else
    {
        std::cout << (-B + sqrt(delta)) / (2 * A) << " " 
                  << (-B - sqrt(delta)) / (2 * A) << std::endl;
    }
    return 0;
}